wkcuber/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .cubing import cubing
 from .downsampling import downsample_mag
 from .compress import compress_mag
-from .metadata import write_webknossos_metadata
+from .metadata import write_webknossos_metadata, read_metadata_for_layer
```

Comment:
Why are these references required? Couldn't you access the function like this `wkcuber.metadata.read_metadata_for_layer`?